theme_test_custo/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    'name': 'Custom Theme (Testing suite)',
+    'summary': '',
+    'description': '',
+    # Remove the `/Hidden` part to make it selectable for tests purpose
```

Comment:
Note that for runbot, you obviously can't change this, so one way is to:
- Go to the `act_window` `website.theme_install_kanban_action` (`Pick a theme`)
- Change the `domain` field to keep only the `[('category_id.parent_id', '=', xx)]` part

Then you should be able to select that theme in the theme selection screen